containers/init/ready/ready.go

```diff
@@ -70,21 +75,9 @@ type PodLister interface {
 	List(metav1.ListOptions) (*corev1.PodList, error)
 }
 
-// parseSelectors accepts a slice of strings and converts them into Kubernetes
-// selectors. It returns an error if any of the strings is not a valid selector.
-func parseSelectors(sels []string) ([]labels.Selector, error) {
-	var selectors []labels.Selector
-
-	for i, arg := range sels {
-		selector, err := labels.Parse(arg)
-		if err != nil {
-			return nil, errors.Wrapf(err, "failed to parse selector #%d (%s): %v", i+1, arg, err)
-		}
-
-		selectors = append(selectors, selector)
-	}
-
-	return selectors, nil
+// LoadtestGetter fetch a load test with a specific name.
+type LoadtestGetter interface {
```

Comment:
nit: Since the object is a `LoadTest`, this should be `LoadTestGetter`. 

```suggestion
type LoadTestGetter interface {
```

Reply:
Done